app.js

```diff
@@ -0,0 +1,88 @@
+const storageKey = 'awesomeBooks';
+const currentFormTitle = document.querySelector('#title');
+const currentFormAuthor = document.querySelector('#author');
+const currentFormBookAddButton = document.querySelector('#book-add-button');
+const arrBooks = [];
+
+function removeBook(id) {
+  for (let i = 0; i < arrBooks.length; i += 1) {
+    if (arrBooks[i].id === id) {
+      arrBooks.splice(i, 1);
+    }
+  }
+}
+
+function addBook() {
+  let maxId = 0;
+  arrBooks.forEach((book) => {
+    if (book.id > maxId) {
+      maxId = book.id;
+    }
+  });
+
+  const newBook = {
+    id: maxId + 1,
```

Comment:
- [Optional] As you did hard work and implemented each and every requirement was asked. I kindly suggest you to use  a unique `id` property for each book since it would make it easier to remove books with duplicate values. [for more info](https://dev.to/rahmanfadhil/how-to-generate-unique-id-in-javascript-1b13)